package.json

```diff
@@ -32,7 +32,7 @@
   },
   "dependencies": {
     "bugsnag": "^1.9.1",
-    "discordie": "^0.10.0",
+    "discordie": "github:qeled/discordie#dev",
```

Comment:
Then perhaps we should stay on a release unless we need something out of the develop branch for discordie. If there is something we put in later that needs that branch we can change it then.